README.md

```diff
@@ -0,0 +1,78 @@
+# mattermost-github-integration
+
+This role sets up a Mattermost GitHub integration service which posts changes of an organization or repository to a mattermost channel.
+For more information see [softdevteams repository](https://github.com/softdevteam/mattermost-github-integration)
+
+## Requirements
+
+This role requires an apt based system like Ubuntu.
+
+
+## Role Variables
+
+| Name                                           |         Required         | Default                              | Description                                                         |
+|:-----------------------------------------------|:------------------------:|:-------------------------------------|:--------------------------------------------------------------------|
+| `global_cache_dir`                             |    :heavy_check_mark:    |                                      | Cache directory to download roundcube files to                      |
+| `mattermost_github_integration_install_path`   | :heavy_multiplication_x: | `/opt/mattermost-github-integration` | Path to install the server on                                       |
+| `mattermost_github_integration_secret`         | :heavy_multiplication_x: | `None`                               | Secret that is used to authenticate between GitHub and your server. |
```

Comment:
So now you add dots in the last column? Just remove them

Reply:
fixed